test/hcl-test.js

```diff
@@ -13,17 +13,17 @@ tape("hcl(…) returns an instance of hcl and color", function(test) {
 });
 
 tape("hcl(…) exposes h, c, and l channel values", function(test) {
-  test.hclEqual(color.hcl("#abc"), 257.7177616818892, 10.774886733325554, 75.10497524893663, 1);
+  test.hclEqual(color.hcl("#abc"), 252.38452899046413, 11.222753486232566, 74.96879980931759, 1);
   test.end();
 });
 
 tape("hcl(…) returns defined hue and chroma, even for black and white", function(test) {
   test.hclEqual(color.hcl("black"), 0, 0, 0, 1);
   test.hclEqual(color.hcl("#000"), 0, 0, 0, 1);
   test.hclEqual(color.hcl(color.lab("#000")), 0, 0, 0, 1);
-  test.hclEqual(color.hcl("white"), 158.19859051364818, 0.00001795054880958058, 100.00000386666655, 1);
-  test.hclEqual(color.hcl("#fff"), 158.19859051364818, 0.00001795054880958058, 100.00000386666655, 1);
-  test.hclEqual(color.hcl(color.lab("#fff")), 158.19859051364818, 0.00001795054880958058, 100.00000386666655, 1);
+  test.hclEqual(color.hcl("white"), 0, 0.003457073518231546, 100, 1);
```

Comment:
Since white is `1 0 0` in Lab, `0` is the appropriate hue in HCL (`Math.atan2(b,a)`).

Reply:
White is 100 0 0 in Lab